shared/js/trackers.js

```diff
@@ -17,266 +17,263 @@ settings.ready().then(() => {
+        if (social_block) {
+            blockSettings.push('Social')
+        }
 
-            var whitelistedTracker = checkWhitelist(urlToCheck, siteDomain, request)
-            if (whitelistedTracker) {
-                return whitelistedTracker
-            }
+        var whitelistedTracker = checkWhitelist(urlToCheck, siteDomain, request)
+        if (whitelistedTracker) {
+            return whitelistedTracker
+        }
 
 
-            var surrogateTracker = checkSurrogateList(urlToCheck, parsedUrl, currLocation)
-            if (surrogateTracker) {
-                return surrogateTracker 
-            }
-
-            // Look up trackers by parent company. This function also checks to see if the poential
-            // tracker is related to the current site. If this is the case we consider it to be the
-            // same as a first party requrest and return
-            var trackerByParentCompany = checkTrackersWithParentCompany(blockSettings, urlSplit, currLocation)
-            if (trackerByParentCompany) {
-            // check cancel to see if this tracker is related to the current site
-                if (trackerByParentCompany.cancel) {
-                    return
-                } else {
-                    return trackerByParentCompany
-                }
-            }
+        var surrogateTracker = checkSurrogateList(urlToCheck, parsedUrl, currLocation)
+        if (surrogateTracker) {
+            return surrogateTracker 
+        }
 
-            // block trackers from easylists
-            let easylistBlock = checkEasylists(urlToCheck, siteDomain, request)
-            if (easylistBlock) {
-                return easylistBlock
+        // Look up trackers by parent company. This function also checks to see if the poential
+        // tracker is related to the current site. If this is the case we consider it to be the
+        // same as a first party requrest and return
+        var trackerByParentCompany = checkTrackersWithParentCompany(blockSettings, urlSplit, currLocation)
+        if (trackerByParentCompany) {
+        // check cancel to see if this tracker is related to the current site
+            if (trackerByParentCompany.cancel) {
+                return
+            } else {
+                return trackerByParentCompany
             }
+        }
 
+        // block trackers from easylists
+        let easylistBlock = checkEasylists(urlToCheck, siteDomain, request)
+        if (easylistBlock) {
+            return easylistBlock
         }
-        return false
-    }
 
-    function checkWhitelist(url, currLocation, request) {
-        let result = false
-        let match
-        const whitelists = abpLists.getWhitelists()
+    }
+    return false
+}
 
-        if (whitelists.trackersWhitelist.isLoaded) {
-            match = checkABPParsedList(whitelists.trackersWhitelist.parsed, url, currLocation, request)
-        }
+function checkWhitelist(url, currLocation, request) {
+    let result = false
```

Comment:
`result` was used further down but not defined. It didn't seem like a global, so I just defined here.